components/ActivityIndicator/examples/ActivityIndicatorExample.swift

```diff
@@ -0,0 +1,90 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+
+import UIKit
+import MaterialComponents
+
+struct MDPalette {
+   static let blue: UIColor = UIColor(red: 0.129, green: 0.588, blue: 0.953, alpha: 1.0)
+   static let red: UIColor = UIColor(red: 0.957, green: 0.263, blue: 0.212, alpha: 1.0)
+   static let green: UIColor = UIColor(red: 0.298, green: 0.686, blue: 0.314, alpha: 1.0)
+   static let yellow: UIColor = UIColor(red: 1.0, green: 0.922, blue: 0.231, alpha: 1.0)
+}
+
+class ActivityIndicatorSwiftController: UIViewController {
+   
+   override func viewDidLoad() {
+      super.viewDidLoad()
+      // Do any additional setup after loading the view, typically from a nib.
+      
+      
+      //Initialize single color progress indicator
+      let frame1: CGRect = CGRect(x: view.bounds.width / 2 - 16, y: view.bounds.height / 2 - 116, width: 32, height: 32)
```

Comment:
Don't worry about this PR, but in the future I think readability would be improved if these constants were pulled out or derived in some way.